mungegithub/github/github.go

```diff
@@ -2055,7 +2058,11 @@ func (obj *MungeObject) MergePR(who string) bool {
 	if config.DryRun {
 		return true
 	}
-	mergeBody := fmt.Sprintf("Automatic merge from %s", who)
+	mergeBody := fmt.Sprintf("Automatic merge from %s.", who)
+	isMaster, _ := obj.IsForBranch(branchMaster)
+	if config.Project == projectKubernetes && isMaster {
```

Comment:
Instead of hardcoding a check to see if the repo is `kubernetes` we should just make the merge message configurable. Like so:
```
mergeBody := fmt.Sprintf("Automatic merge from %s. %s", who, mergeMessage)
```
Then in the submit-queue code, before calling `MergePR`, check if the pr is for the master branch. If it is, specify a merge message taken from the config (a config option named something like `merge-to-master-message`), if it isn't for master, provide the empty string as the mergeMessage. We can then set a `merge-to-master-message` config option to `If you want to cherry-pick this change to another branch, please follow the instructions <a href=\"https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md\">here</a>.` for the kubernetes repo and we can leave it blank elsewhere.

This will make the message optional and configurable and avoids hard coding specific behavior for a certain repo.